cvss40.js

```diff
@@ -437,13 +446,18 @@ class Vector {
      */
     updateMetricsFromVectorString(vector) {
         if (!vector) {
-            throw new Error("The vector string cannot be null, undefined, or empty.");
+            throw new Error(`The vector string cannot be null, undefined, or empty. in ${vector}`);
```

Comment:
Full stop in the middle of the sentence.

Reply:
Good catch, thank you.